sample-queries/sample-queries.json

```diff
@@ -1426,6 +1426,23 @@
         "tip": "This query requires your user id and another user's id.  To find your user id & another user's id, you can run: 1) GET https://graph.microsoft.com/v1.0/me/ 2) GET https://graph.microsoft.com/v1.0/users",
         "skipTest": false
     },
+    {
+        "id": "8cd7a323-478e-4828-878f-7e6962728a9b",
+        "category": "Microsoft Teams",
+        "method": "POST",
+        "humanName": "send activity notification",
```

Comment:
There are 3 different types of activity notifications and they are all under Teams. It may be a good idea to clarify which one you are using in the humanName. I looked at the other queries and there seemed to be longer human names so I think it should be okay to make it longer. Just a trade off for clarity